lib/rubocop/cop/style/trailing_body_on_class_definition.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Style
+      # This cop checks for trailing code after the class definition.
+      #
+      # @example
+      #   # bad
+      #   Class Foo; def foo; end
```

Comment:
Class -> class

Reply:
fixed in the final commit

Reply:
I will clean it up.  But want to get your thoughts on Cop names, and any other naming in here beforehand.